src/nvim/ex_getln.c

```diff
@@ -906,6 +906,7 @@ static uint8_t *command_line_enter(int firstc, long count, int indent)
     }
     tl_ret = true;
   }
+  trigger_modechanged();
```

Comment:
Patch v8.2.3430 had

```c
if (!debug_mode)
  trigger_modechanged();
```

I guess that `debug_mode` should not be in the vim patch. It's rarely used throughout the codebase.

Reply:
The `debug_mode` is only there, because some weird behaviour in some test caused the test to fail without it. See https://github.com/vim/vim/pull/8856#issuecomment-916805623 for reference. This is not the case for neovim, so I exclude `debug_mode` in the port.